recipes-nodes/config-watchdog/config-watchdog.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/sh
+### BEGIN INIT INFO
+# Provides:          config-watchdog
+# Required-Start:    $network
+# Required-Stop:
+# Default-Start:     2 3 4 5
+# Default-Stop:      0 1 6
+# Short-Description: config watchdog
+# Description:       Pulls the config from BuilderHub and reloads repsective services
+### END INIT INFO
+
+set -e
+
+SYSTEM_API_FIFO="/var/volatile/system-api.fifo"
+RBUILDER_PERSISTENT_DIR="/persistent/rbuilder"
+RBUILDER_STATUS_FLAG_FILE="${RBUILDER_PERSISTENT_DIR}/rbuilder.enabled"
+# locally exposed BuilderHub API proxied through CVM proxy
+BHUB_URL_PROXIED="http://localhost:7937/api/l1-builder/v1/configuration"
+BHUB_CONFIG_FILE="/etc/init-config-watchdog.json"
+
+function cleanup() {
+  rm -f "${BHUB_CONFIG_FILE}"
+}
+
+trap cleanup EXIT
+
+log() {
+    if [ -p $SYSTEM_API_FIFO ]; then
+        date_log() {
+            echo -n ""
+        }
+    else
+        date_log() {
+            echo -n "$(date --iso-8601=seconds): "
+        }
+    fi
+    echo "$(date_log)$1" | tee -a $SYSTEM_API_FIFO
+}
+
+function toggle_rbuilder() {
```

Comment:
toggle implies it always switches from one to the other. i'd suggest to give it a clearer name, like

```suggestion
function check_rbuilder_enabled() {
```